packages/legend-application-query/src/components/QueryBuilderProjectionPanel.tsx

```diff
@@ -620,10 +639,17 @@ export const QueryBuilderProjectionPanel = observer(
 
     return (
       <div
-        className="panel__content dnd__overlay__container"
         ref={dropConnector}
+        className={`panel__content dnd__overlay__container`}
       >
+        <div className={clsx({ projection__error: isInvalidProjection })}>
+          {projectionState.getValidationErrorMessage() && (
```

Comment:
Let's render the error anyway, but then we can do the message as `projectionState.getValidationErrorMessage() ?? ''`